stable/drone/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 1.7.4
-appVersion: 0.8.6
+version: 2.0.0
+appVersion: 1.0.0-rc.2
```

Comment:
I recommend holding until 1.0.0 releases

Reply:
I had assumed that rc.1 would follow release candidate patterns. But rc.2 has made it clear, we may be in for a bumpy ride, to final.  So, I'll likely close the PR, as it's hard to get consensus on a moving target. I'll keep updating my branch, on my own, because I truly hate managing yaml files.